### ISSUE TITLE
Running a describe skips all the tests

### ISSUE THREAD

**Issue by matdurand** [firsttris/vscode-jest-runner#51]
Let's take this example:
```
describe('test1', () => {
    it('should run this test', () => {
      expect(true).toBe(true);
    });
});
```

If I place my cursor on "test1", the describe statement, and run jest, it will run with `-t '^test1$'` which I think is wrong since it doesnt match any test, because of the $. No tests are ending with "test1". Maybe you should only add the $ when the cursor is on a specific test and not on a describe.

**Comment by firsttris** [comment]
Yes i thought about this. There ist still the option that you Highlight the text an click. Then it would not do the exact match. But maybe i could still improve this

**Comment by andypearson** [comment]
Running a `describe` block with my keyboard shortcut was a big part of my workflow so I'm sad to see this gone :(

Could something be done to treat a line that includes `describe` differently to a line that starts with `it` or `test`?

I've rolled back to `0.4.2` to get this functionality back for the time being.

**Comment by firsttris** [comment]
@andypearson just note that you can still highlight/mark the text inside the describe and it will run describe correctly in 0.4.9.

i will fix this asap

**Comment by firsttris** [comment]
please verify and report back if your issue is solved with 0.4.12

**Comment by RandomEngy** [comment]
Now nothing works: I get this error:

cd "c:\gitrepo&& node "c:/gitrepo/node_modules/jest/bin/jest.js" "c:/gitrepo/experiences/pivots-nav/src/PivotsNav.connector.spec.ts" -t "PivotsNavConnector"
At line:1 char:33
+ cd "c:\gitrepo" && node "c:/gitrepo/n ...
+                                 ~~
The token '&&' is not a valid statement separator in this version.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : InvalidEndOfLine

**Comment by firsttris** [comment]
fixed it with 0.4.15. it seems && is supported by cmd, bash, fish but not powershell. its harder then you think to support all terminals also considering multi workspace setups and mono repos.

feel free to re-open or create a new issue if you find one.

**Comment by RandomEngy** [comment]
0.4.15 works! Dealing with all the different terminals sounds annoying. Thanks for the fix.
